sections/shared/Layout/AppLayout/Header/BalanceActions.tsx

```diff
@@ -75,6 +69,8 @@ const BalanceActions: FC = () => {
 		}
 	}, [dispatch, walletAddress]);
 
+	usePollAction('fetchEarnTokenPrices', fetchEarnTokenPrices, { intervalTime: 30000 });
```

Comment:
I think now we've added this here and it will be called on every page we can remove the other same poll in `StakeGrid`

Reply:
Nice catch. I will change this.